test/test_road_distance.py

```diff
@@ -0,0 +1,42 @@
+import os
+from pandas import DataFrame,Series,cut
+from pandas.testing import assert_frame_equal, assert_series_equal
+from pytest import fixture
+
+
+# class TestSettlementProcessor:
+
+#     @fixture
+#     def setup_settlementprocessor(self) -> SettlementProcessor:
+
+#         csv_path = os.path.join('test', 'test_data', 'dj-test.csv')
+#         settlementprocessor = SettlementProcessor(csv_path)
+
+#         return settlementprocessor
+
+@fixture
+def setup_dataframe() -> DataFrame:
+    df = DataFrame({'RoadDist':[0,11.954,14.282,10.472]})
+    return df
+   
+def test_road_distance(setup_dataframe):
```

Comment:
So, this test doesn't actually test the onsset code...  If you create a new `classify_road_distance` method as mentioned above, then you could replace lines 33-34 with the call to this method.